Library/Formula/ddar.rb

```diff
@@ -8,11 +8,14 @@ class Ddar < Formula
   head "https://github.com/basak/ddar.git"
 
   depends_on "xmltoman" => :build
-  depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on :python
```

Comment:
Tim might throw me into something if I get this wrong yet again, but:

* `depends_on :python if MacOS.version <= :snow_leopard` = Don't use Homebrew's Python unless building for Snow Leopard or below.
* `depends_on :python` = Use the first Python in the `$PATH` for `--build-from-source` but bottles will be built against Homebrew's Python.
* Lack of `depends_on :python` but still uses Python = Always use the system. Software is compatible with Python 2.6 on old platforms.

Due to our `default_formula` mechanism, `depends_on :python` builds the bottle against Homebrew's Python and consequently forces the installation of our Python when the bottle is used.